featurizers/fingerprints.py

```diff
@@ -261,3 +275,94 @@ def _calculate_fingerprint(
         }
 
         return np.array([GetErGFingerprint(x, **fp_args) for x in X])
+
+
+class E3FP(FingerprintTransformer):
+    def __init__(
+        self,
+        bits: int,
+        radius_multiplier: float,
+        rdkit_invariants: bool = True,
+        first: int = 1,
+        num_conf: int = NUM_CONF_DEF,
+        pool_multiplier: float = POOL_MULTIPLIER_DEF,
+        rmsd_cutoff: float = RMSD_CUTOFF_DEF,
+        max_energy_diff: float = MAX_ENERGY_DIFF_DEF,
+        force_field: float = FORCEFIELD_DEF,
+        get_values: bool = True,
+        is_folded: bool = False,
+        fold_bits: int = 1024,
+        standardise: bool = True,
```

Comment:
I still prefer "standardize" (https://dictionary.cambridge.org/dictionary/english/standardize), since US English is more popular for programming, but ok